src/LightGraphs.jl

```diff
@@ -148,6 +149,7 @@ include("core.jl")
     const Edge = SimpleGraphs.SimpleEdge
 
     include("digraph-transitivity.jl")
+    include("digraph-cyclicity.jl")
```

Comment:
Does it make sense to split this up into a directory of files? That is, `digraph/cyclicity/hadwick-james.jl`, `digraph/cyclicity/...`?